redhat-director-scripts/docker/build_rhosp15.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -x
```

Comment:
if we have added code to create rhosp15 containers in **build_containers.sh** then do we really need to have seperate file **(build_rhosp15.sh)** which does same task.

Reply:
This is dev testing purpose.